pallets/gear/src/manager/task.rs

```diff
@@ -41,8 +41,19 @@ impl<T: Config> TaskHandler<T::AccountId> for ExtManager<T>
 where
     T::AccountId: Origin,
 {
-    fn pause_program(&mut self, _program_id: ProgramId) {
-        todo!("#646");
+    fn pause_program(&mut self, program_id: ProgramId) {
```

Comment:
I'd set enormous free period for now

Reply:
to avoid expiring of programs without resume functionality

Reply:
agree